Kajabi/sage-lib
# List: Sortable list interactions update

## Description

This PR updates styling for interactions with sortable List.

## Screenshots

[Figma spec](https://www.figma.com/file/sMbtLUHSt2vfKgKjyQ3052/Sage-components?node-id=12501%3A23675)

| Before | After |
|---|---|
| ![2022-08-01 16 34 24](https://user-images.githubusercontent.com/17955295/182242290-81bf71c6-0f75-4786-b5fc-30a2d5d273f0.gif) | ![2022-08-01 16 31 30](https://user-images.githubusercontent.com/17955295/182242199-e50579b4-9ed1-41a7-bc9e-8eedc8d78f56.gif) |

## Testing in `sage-lib`

- See Rails http://localhost:4000/pages/component/list?tab=preview
- See React http://localhost:4100/?path=/story/sage-list--sortable-list

## Testing in `kajabi-products`

1. (LOW) Updates interactive styling for the List (sortable) component in Rails and React.

## Related

[SAGE-535](https://kajabi.atlassian.net/browse/SAGE-535)
